INSTALL.md

```diff
@@ -190,13 +190,28 @@ installed as well as [Clang 3.9 or greater](http://releases.llvm.org/download.ht
 
 #### Clear Linux
 
-On Clear Linux, you need a few extra libraries to build Alacritty. Here's a
-`swupd` command that should install all of them. If something is still found
-to be missing, please open an issue.
+On Clear Linux, a few extra steps are needed to build Alacritty. 
+
+1. Here's a `swupd` command that should install all missing dependencies. If something is still found to be missing, please open an issue.
 
 ```sh
-swupd bundle-add devpkg-expat devpkg-freetype devpkg-libxcb
+swupd bundle-add devpkg-fontconfig devpkg-libxcb
+```
+
+2. Clear Linux stores fonts.conf in a non-standard location due to its [stateless](https://github.com/clearlinux/clr-man-pages/blob/master/stateless.7.rst) design. 
+
+* Either set the FONTCONFIG_PATH environment variable as follows:
+
 ```
+echo "FONTCONFIG_PATH=/usr/share/defaults/fonts" >> /etc/profile
+```
+
+* or create a symbolic link to the location of fonts.conf:
```

Comment:
I'm not sure about bullet list here tbh. What do you think @chrisduerr ?

Reply:
Is either method less likely to cause issues for other programs? I agree the list adds ambiguity, I just didn't know which one would be better to recommend.

Reply:
I'm taking about markdown style issue. using `*` creates a bullet list, so I don't really like how it looks like tbh.

Reply:
The entire section looks a bit off, since it's all written as a single big list, but that doesn't really make much sense.

The `Here's a swup command` comment does not fit in with the rest of the documentation and thus should be adjusted. `The following swup command` would be more appropriate.

The entire numbering and sub-lists just create unnecessary indentation that make the text hard to read. We're already listing a bunch of things that all need to be executed and there is no ambiguity about the order of operations, so we don't need to use lists for this.

It also seems like this is something that should be fixed in clear linux, not Alacritty. Even with their 'stateless design', it pretty clearly seems broken? We should be fine just recommending the symlink though, I don't think exporting environment variables is following the goals of the Clear Linux project?

Reply:
When [an issue](https://github.com/clearlinux/distribution/issues/639) was opened with Clear, one of their devs said 
> Does it search for that file directly, not using libfontconfig? If so, it needs a patch to operate properly on Clear.

I'm not taking a side on that, just sharing in case it is a more general solution.

Reply:
> Does it search for that file directly, not using libfontconfig

I'm pretty sure the answer to that should be `no`. We call `FcConfigGetCurrent` to get the config, we don't load the file ourselves.

This seems like something that should be fixed, rather than worked around.

Reply:
I feel like Clear linux passes this thing internally to bundles apps or something like that, since entire /etc is empty on it IIRC. So I adding such things into /etc/profile is a way to go for users and even a recommended way. I could be wrong about "recommended", but I heard something like that in the past.

Reply:
and Alacritty is just calling to `libfontconfig.so` APIs, it doens't do any parsing of config files itself, etc, etc.

Reply:
> So I adding such things into /etc/profile is a way to go for users and even a recommended way. I could be wrong about "recommended", but I heard something like that in the past.

It seems like that should be done for the system then. The comment on the issue makes it sound like the distro is expected to work by default, which it clearly does not then.

So it seems to me like clear linux has got some figuring out to do? Otherwise I'm very curious what their recommended strategy would be?